src/coreComponents/constitutive/permeability/PermeabilityBase.cpp

```diff
@@ -44,6 +44,20 @@ PermeabilityBase::deliverClone( string const & name,
   return ConstitutiveBase::deliverClone( name, parent );
 }
 
+void PermeabilityBase::scaleHorizontalPermeability( arrayView1d< real64 const > scalingFactors ) const
+{
+  localIndex const numElems = m_permeability.size( 0 );
+  for( localIndex ei = 0; ei < numElems; ++ei )
+  {
+    // NOTE: enforcing 1 quadrature point
```

Comment:
The permeability models are all implemented with one quadrature point because there is really no need to use the default behavior (number of quadrature points decided by the finite-element scheme in coupled simulations). This is done in `allocateConstitutiveData` below:
https://github.com/GEOSX/GEOSX/blob/d90763008ffda699a0fdcd094e43ed69da9626c4/src/coreComponents/constitutive/permeability/PermeabilityBase.cpp#L64-L66

So in the new code, I just followed this. After your comment, I modified the new loop to have something cleaner in case someone decides to increase the number of quadrature points:
https://github.com/GEOSX/GEOSX/blob/e5c09cc8da5d3029e044be3c202e5cf65d3ce597/src/coreComponents/constitutive/permeability/PermeabilityBase.cpp#L51-L58 
where `numQuad` is the second dimension of `m_permeability`